<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegressionLogLinearForm</class>
 <widget class="QWidget" name="RegressionLogLinearForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>999</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="topWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0" rowspan="6">
       <widget class="AvailableFieldsListView" name="listAvailableFields">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="boundSingleItemView" name="counts">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Counts (optional)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="AssignButton" name="buttonAssignCounts">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Factors</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widget_9" native="true"/>
      </item>
      <item row="4" column="1">
       <widget class="AssignButton" name="buttonAssignFactors">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2" rowspan="2">
       <widget class="BoundListView" name="factors">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>160</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>160</height>
         </size>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>84</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="modelContainer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="ExpanderButton" name="expanderModel">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="AnovaModelWidget" name="modelTerms" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QGridLayout" name="gridLayout_16">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="ExpanderButton" name="pushButton">
        <property name="text">
         <string>Statistics</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="panelStatistics" native="true">
        <layout class="QGridLayout" name="gridLayout_11">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="1" rowspan="2">
          <widget class="QWidget" name="widget_20" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="BoundCheckBox" name="VovkSellkeMPR">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Vovk-Sellke Maximum p-Ratio</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="GroupBox" name="groupBox">
           <property name="title">
            <string>Regression Coefficients</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_51">
            <item row="2" column="0">
             <widget class="QWidget" name="panelConfidenceInterval" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_18">
               <property name="leftMargin">
                <number>20</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Interval</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="BoundTextBox" name="regressionCoefficientsConfidenceIntervalsInterval">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>%</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="BoundCheckBox" name="regressionCoefficientsConfidenceIntervals">
              <property name="text">
               <string>Confidence intervals</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="BoundCheckBox" name="regressionCoefficientsEstimates">
              <property name="text">
               <string>Estimates</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
  <customwidget>
   <class>BoundListView</class>
   <extends>QListView</extends>
   <header>widgets/boundlistview.h</header>
  </customwidget>
  <customwidget>
   <class>BoundTextBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/boundtextbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>widgets/boundcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/groupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>boundSingleItemView</class>
   <extends>QListView</extends>
   <header>widgets/boundsingleitemview.h</header>
  </customwidget>
  <customwidget>
   <class>ExpanderButton</class>
   <extends>QPushButton</extends>
   <header>widgets/expanderbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AnovaModelWidget</class>
   <extends>QWidget</extends>
   <header>widgets/anovamodelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>regressionCoefficientsConfidenceIntervals</sender>
   <signal>toggled(bool)</signal>
   <receiver>panelConfidenceInterval</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>608</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>637</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
